util/reconciler/defaults.go

```diff
@@ -22,9 +22,15 @@ import (
 
 const (
 	// DefaultLoopTimeout is the default timeout for a reconcile loop (defaulted to the max ARM template duration).
-	DefaultLoopTimeout = 90 * time.Minute
+	DefaultLoopTimeout = 1 * time.Minute
```

Comment:
// TODO: might need to change this back given that we don't have async enabled everywhere yet